logicrunner/goplugin/goplugin.go

```diff
@@ -165,6 +167,8 @@ func (gp *GoPlugin) CallMethod(
 
 	select {
 	case callResult := <-resultChan:
+		callTime := time.Since(start)
+		metrics.GopluginContractExecutionTime.Observe(callTime.Seconds())
 		inslogger.FromContext(ctx).Debugf("CallMethod done work, time spend in here - %s", time.Since(start))
```

Comment:
here you should probably re-use callTime variable in the logging

Reply:
fixed